.doc_gen/metadata/cross_metadata.yaml

```diff
@@ -1,4 +1,16 @@
 # zexi 0.2.0
+cross_SnsPublishSubscription:
+  title: Creating an &SNSlong; Publish and Subscription Web Application that Translates Messages
+  title_abbrev: Building an &SNS; Sub Web Application 
```

Comment:
Build a publish and subscription web application that translates messages
* Imperative verb
* No entities in title_abbrev (usually--your call on this one)
* Sentence case

Reply:
fixed